Ofir.yefet/add api catalog documentation (#21899)
DataDog/documentation: content/en/api_catalog/troubleshoot.md

## Missing endpoints

> API Catalog is based on APM tracing, so the first step is to make sure your services are instrumented by hitting the 'learn more' button in the app.

Reviewer: ```suggestion
API Catalog is based on APM tracing, so the first step is to make sure your services are instrumented. Click **Learn More** in the app, then select **Troubleshoot** to verify this.
```

> {{< img src="tracing/api_catalog/api-catalog-discovery-learn-more.png" alt="ALT TEXT" style="width:30%;text-align: left;" >}}

Reviewer: ```suggestion
{{< img src="tracing/api_catalog/api-catalog-discovery-learn-more.png" alt="Learn More button in the app" style="width:30%;text-align: left;" >}}
```

## No data for new monitor

API Catalog is based on APM tracing, so info from traffic is displayed only when there are traces for the endpoint. If you do not see data in the monitor graph, one of the options below may apply:

> - The endpoint has not been used yet since it was just registered/uploaded through OpenAPI.

Reviewer: ```suggestion
- The endpoint has not been used yet since it was registered/uploaded through OpenAPI.
```
^ Making our Vale linter happy 😄
